src/stats/stats-service-openmetrics.c

```diff
@@ -210,7 +210,7 @@ openmetrics_export_histogram(struct openmetrics_request *req, string_t *out,
 			openmetrics_find_histogram_bucket(metric, i);
 
 		if (sub_metric != NULL) {
-			sum += stats_dist_get_sum(sub_metric->duration_stats);
+			sum += stats_dist_get_sum(sub_metric->duration_stats)/1000.0;
```

Comment:
why is this being divided by 1000? also these divisions should be in separate commit.

Reply:
I can add a comment to it, why it's converted.